commit df34df483a97b1591a3e90a6941f99fe9f863508
Merge: 9abf8acea297 049b5e2ae30b
Author: Linus Torvalds <torvalds@linux-foundation.org>
Date:   Wed Apr 4 18:56:27 2018 -0700

    Merge tag 'staging-4.17-rc1' of git://git.kernel.org/pub/scm/linux/kernel/git/gregkh/staging
    
    Pull staging/IIO updates from Greg KH:
     "Here is the big set of Staging/IIO driver patches for 4.17-rc1.
    
      It is a lot, over 500 changes, but not huge by previous kernel release
      standards. We deleted more lines than we added again (27k added vs.
      91k remvoed), thanks to finally being able to delete the IRDA drivers
      and networking code.
    
      We also deleted the ccree crypto driver, but that's coming back in
      through the crypto tree to you, in a much cleaned-up form.
    
      Added this round is at lot of "mt7621" device support, which is for an
      embedded device that Neil Brown cares about, and of course a handful
      of new IIO drivers as well.
    
      And finally, the fsl-mc core code moved out of the staging tree to the
      "real" part of the kernel, which is nice to see happen as well.
    
      Full details are in the shortlog, which has all of the tiny cleanup
      patches described.
    
      All of these have been in linux-next for a while with no reported
      issues"
    
    * tag 'staging-4.17-rc1' of git://git.kernel.org/pub/scm/linux/kernel/git/gregkh/staging: (579 commits)
      staging: rtl8723bs: Remove yield call, replace with cond_resched()
      staging: rtl8723bs: Replace yield() call with cond_resched()
      staging: rtl8723bs: Remove unecessary newlines from 'odm.h'.
      staging: rtl8723bs: Rework 'struct _ODM_Phy_Status_Info_' coding style.
      staging: rtl8723bs: Rework 'struct _ODM_Per_Pkt_Info_' coding style.
      staging: rtl8723bs: Replace NULL pointer comparison with '!'.
      staging: rtl8723bs: Factor out rtl8723bs_recv_tasklet() sections.
      staging: rtl8723bs: Fix function signature that goes over 80 characters.
      staging: rtl8723bs: Fix lines too long in update_recvframe_attrib().
      staging: rtl8723bs: Remove unnecessary blank lines in 'rtl8723bs_recv.c'.
      staging: rtl8723bs: Change camel case to snake case in 'rtl8723bs_recv.c'.
      staging: rtl8723bs: Add missing braces in else statement.
      staging: rtl8723bs: Add spaces around ternary operators.
      staging: rtl8723bs: Fix lines with trailing open parentheses.
      staging: rtl8723bs: Remove unnecessary length #define's.
      staging: rtl8723bs: Fix IEEE80211 authentication algorithm constants.
      staging: rtl8723bs: Fix alignment in rtw_wx_set_auth().
      staging: rtl8723bs: Remove braces from single statement conditionals.
      staging: rtl8723bs: Remove unecessary braces from switch statement.
      staging: rtl8723bs: Fix newlines in rtw_wx_set_auth().
      ...

diff --cc MAINTAINERS
index f01147af694b,34151f4960c9..e56cedcd01a0
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@@ -8592,17 -8600,9 +8598,18 @@@ M:	Peter Rosin <peda@axentia.se
  L:	linux-iio@vger.kernel.org
  S:	Maintained
  F:	Documentation/ABI/testing/sysfs-bus-iio-potentiometer-mcp4531
+ F:	drivers/iio/potentiometer/mcp4018.c
  F:	drivers/iio/potentiometer/mcp4531.c
  
 +MCR20A IEEE-802.15.4 RADIO DRIVER
 +M:	Xue Liu <liuxuenetmail@gmail.com>
 +L:	linux-wpan@vger.kernel.org
 +W:	https://github.com/xueliu/mcr20a-linux
 +S:	Maintained
 +F:	drivers/net/ieee802154/mcr20a.c
 +F:	drivers/net/ieee802154/mcr20a.h
 +F:	Documentation/devicetree/bindings/net/ieee802154/mcr20a.txt
 +
  MEASUREMENT COMPUTING CIO-DAC IIO DRIVER
  M:	William Breathitt Gray <vilhelm.gray@gmail.com>
  L:	linux-iio@vger.kernel.org